content/hardware/07.opta/opta-family/opta/tech-specs.yml

```
@@ -8,6 +8,7 @@ Microcontroller:
   Cortex-M7 core: Up to 480 MHz
   Cortex-M4 core: Up to 240 MHz
 Input:
+  Max input Voltage: 10V
```

```suggestion
  Max input Voltage: 10 V
```

It's a nice addition.

Changed to separate the 10V to 10 V to keep consistency